mayastor/src/bdev/nexus/nexus_fn_table.rs

```diff
@@ -52,16 +52,16 @@ impl NexusFnTable {
         &NEXUS_FN_TBL.f_tbl
     }
 
-    /// check all the children for the specified IO type and return if it
+    /// check all the children for the specified IO type and return if it is
     /// supported
     extern "C" fn io_supported(
         ctx: *mut c_void,
         io_type: spdk_bdev_io_type,
     ) -> bool {
         let nexus = unsafe { Nexus::from_raw(ctx) };
         match io_type {
-            // we always assume the device supports read/write commands
-            io_type::READ | io_type::WRITE => true,
+            // we always assume the device supports read/write/admin commands
+            io_type::READ | io_type::WRITE | io_type::NVME_ADMIN => true,
```

Comment:
What if one of the children is not shared by nvme ?

Reply:
It won't work but AIUI we're moving towards sharing over NVMe anyway.

Reply:
I think in k8s, as it currently stands, we only ever have nvmf children.
For when you're not running in k8s, maybe we can use that io_supported to check if all children support it?

Reply:
OK, I'll check io_supported.